### PR TITLE
[docs] Proofreading

### PR DESCRIPTION
**Pull request by VV-corrector**
Documentation proofreading.

### CODE REVIEW

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 1]
```diff
@@ -71,15 +71,15 @@ kubectl -n d8-system edit moduleconfig/upmeter
...
-- `spec.settings` — настройки модуля. Необязательное поле, если используется поле `spec.enabled`. Описание возможных настроек можно найти в документации модуля в разделе *"Настройки"*.
-- `spec.enabled` — необязательное поле для явного [включения или отключения модуля](#включение-и-отключение-модуля). Если не задано, модуль может быть включён по умолчанию в одном из [наборов модулей](#наборы-модулей).
+- `metadata.name` — название модуля Deckhouse в kebab-case (например, `prometheus`, `node-manager`);
+- `spec.version` — версия схемы настроек модуля (целое число, больше нуля). Обязательное поле, если `spec.settings` не пустое. Номер актуальной версии можно увидеть в документации модуля в разделе *«Настройки»*:
+  - Deckhouse поддерживает обратную совместимость версий схемы настроек модуля. Если используется схема настроек устаревшей версии, при редактировании или просмотре custom resource'а будет выведено предупреждение о необходимости обновить схему настроек модуля;
+- `spec.settings` — настройки модуля. Необязательное поле, если используется поле `spec.enabled`. Описание возможных настроек можно найти в документации модуля в разделе *«Настройки»*;
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:81]:
```suggestion
- `spec.settings` — настройки модуля. Необязательное поле, если используется поле `spec.enabled`. Описание возможных настроек можно найти в документации модуля в разделе *Настройки*;
```

File: docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md [hunk 2]
```diff
@@ -71,15 +71,15 @@ kubectl -n d8-system edit moduleconfig/upmeter
...
-- `spec.version` — версия схемы настроек модуля (целое число, больше нуля). Обязательное поле, если `spec.settings` не пустое. Номер актуальной версии можно увидеть в документации модуля в разделе *"Настройки"*.
-  - Deckhouse поддерживает обратную совместимость версий схемы настроек модуля. Если используется схема настроек устаревшей версии, при редактировании или просмотре custom resource'а будет выведено предупреждение о необходимости обновить схему настроек модуля.
-- `spec.settings` — настройки модуля. Необязательное поле, если используется поле `spec.enabled`. Описание возможных настроек можно найти в документации модуля в разделе *"Настройки"*.
-- `spec.enabled` — необязательное поле для явного [включения или отключения модуля](#включение-и-отключение-модуля). Если не задано, модуль может быть включён по умолчанию в одном из [наборов модулей](#наборы-модулей).
+- `metadata.name` — название модуля Deckhouse в kebab-case (например, `prometheus`, `node-manager`);
+- `spec.version` — версия схемы настроек модуля (целое число, больше нуля). Обязательное поле, если `spec.settings` не пустое. Номер актуальной версии можно увидеть в документации модуля в разделе *«Настройки»*:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE_CONFIGURE_RU.md:79]:
```suggestion
- `spec.version` — версия схемы настроек модуля (целое число, больше нуля). Обязательное поле, если `spec.settings` не пустое. Номер актуальной версии можно увидеть в документации модуля в разделе *Настройки*:
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [5878](https://github.com/deckhouse/deckhouse/pull/5878) to the branch [release-1.51](https://github.com/deckhouse/deckhouse/tree/release-1.51) successful!

**Pull request merged**